### PR TITLE
Print better logs for apex tests

### PR DESCRIPTION
**Pull request by PatoBeltran**
Apex output was confusing, with this change it will print out in which test case it is running so it is easier to debug.

Also cleaned up targets not used in build.proj

### CODE REVIEW

**Comment by rohit21agrawal** [comment]
🏅  
🚢  once it has run on CI

@PatoBeltran is there a cheap way to get this kind of logging for all of our tests?

**Review by rohit21agrawal — APPROVED** [review]

File: build/build.proj [hunk 1]
```diff
@@ -156,33 +126,8 @@
...
-              ItemName="TestAssemblyPath" />
-    </MSBuild>
-  </Target>
-
-  <!--
-    ============================================================
-    Run VS14 unit tests
-    ============================================================
-  -->
-  <Target Name="UnitTestsVS14">
```
Comment (nkolev92) [inline build/build.proj:170]: savage! 
rip vs14 :D

**Review by nkolev92 — APPROVED** [review]
LGTM

**Comment by PatoBeltran** [comment]
@rohit21agrawal I'm not sure, using the ` "methodDisplay": "classAndMethod"` in the xunit runner might do it, we would need to test it in the different tests

**Pull request merged**